.travis.yml

```diff
@@ -110,6 +110,16 @@ before_install:
 - export ACTIVITI_CLOUD_VERSION=$(grep -oPm1 "(?<=<activiti-cloud.version>)[^<]+"
   "activiti-cloud-dependencies/pom.xml")
 - echo ACTIVITI_CLOUD_VERSION=${ACTIVITI_CLOUD_VERSION}
+- if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then export BRANCH_NAME=${TRAVIS_BRANCH}; else export BRANCH_NAME=PR-${TRAVIS_PULL_REQUEST}; fi
+- if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then echo $(jx-release-version) > VERSION; else echo 7.1.0-${BRANCH_NAME}-${TRAVIS_BUILD_NUMBER} > VERSION; fi
+- export VERSION=$(cat VERSION)
+- echo "VERSION=$VERSION"
+- export PREVIEW_NAMESPACE=$(echo "${BRANCH_NAME}-${TRAVIS_BUILD_NUMBER}" | tr '[:upper:]' '[:lower:]' )
+- echo "PREVIEW_NAMESPACE=$PREVIEW_NAMESPACE"
+- export GATEWAY_HOST="gateway.$PREVIEW_NAMESPACE.$GLOBAL_GATEWAY_DOMAIN" 
+- echo GATEWAY_HOST=$GATEWAY_HOST
+- export SSO_HOST="identity.$PREVIEW_NAMESPACE.$GLOBAL_GATEWAY_DOMAIN" && echo $SSO_HOST
+- echo SSO_HOST=$SSO_HOST
```

Comment:
I think we don't need this here, as it's already configured [here](https://github.com/Activiti/activiti-cloud-application/pull/211/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R2). You can temporarily update it to use the branch where you have the changes.